packages/flutter/test/material/scaffold_test.dart

```diff
@@ -2867,7 +2867,47 @@ void main() {
       expect(tester.takeException(), isNull);
   });
 
-   testWidgets('ScaffoldMessenger showSnackBar default animation', (WidgetTester tester) async {
```

Comment:
I had some trouble trying to see what was changed about this line… thanks for fixing the indent!